src/gmail.js

```diff
@@ -992,6 +992,14 @@ var Gmail = function(localJQuery) {
             && api.check.data.is_email_id(obj["1"]);
     };
 
+    api.check.data.is_action = function(obj) {
+        return obj
+            && Array.isArray(obj)
+            && obj.length === 1
+            && typeof obj["0"] === 'string'
+            && !api.check.data.is_email_id(obj["0"]);
+    };
```

Comment:
I’m still not sure what kind of structure this is. 

Having a unit-test for this function would both help ensure correctness *and* illustrate what kind of data we are working with.

Reply:
ok!)